client/asset/eth/eth.go

```diff
@@ -1392,6 +1398,8 @@ func (w *TokenWallet) FundOrder(ord *asset.Order) (asset.Coins, []dex.Bytes, err
 		}
 	}()
 
+	w.log.Debugf("Locking %s to swap %s in up to %d swaps at a fee rate of %d gwei/gas using up to %d gas per swap",
+		w.parent.amtString(ethToLock), w.amtString(ord.Value), ord.MaxSwapCount, ord.MaxFeeRate, g.Swap)
```

Comment:
`[DBG] CORE[eth][USDC.ETH]: Locking 0.045400000 ETH to swap 46.120000 USDC in up to 1 swaps at a fee rate of 200 gwei/gas using up to 227000 gas per swap`

Super high gas price (200 gwei/gas) there, but informative.